src/App.js

```diff
@@ -1,7 +1,33 @@
+import React, { Component } from "react";
+import ProductDisplay from "./components/ProductDisplay";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+import formatPrice from "./helpers/formatPrice"
 import "./App.css";
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
+export default class App extends Component {
+  state = { cart: []}
 
-export default App;
+  addToCart = (product) => {
+    this.setState((prevState)=>({
+      cart: [...prevState.cart, product]
+    }))
+  }
+  render() {
+    const {cart} = this.state
+    const { products } = this.props;
```

Comment:
I've noticed a few people doing this - you can pass `products` down from `index.js` like you are here, or you could just import the products directly in this file.